src/hooks/api/places.ts

```diff
@@ -309,6 +309,57 @@ const useGetPlacesByQuery = (
     ...configuration,
   });
 
+type GetBPostAddressArguments = {
```

Comment:
I wouldn't use Bpost in the name.
I doesn't really add any value, especially since we don't connect with Bpost directly but via the search api...

If we are switching to another service (instead of bpost) then we also would have to change these names... 

```suggestion
type GetAddressesStreetsArguments = {
```

This is just my opninion, feel free to think differently :)